README.md

```diff
@@ -79,24 +74,29 @@ For example, a script name that follows this convention is: `V1.1.1__first_chang
 
 Every script within a database folder must have a unique version number. snowchange will check for duplicate version numbers and throw an error if it finds any. This helps to ensure that developers who are working in parallel don't accidently (re-)use the same version number.
 
-### Change History Table
+### Script Requirements
+
+snowchange is designed to be very lightweight and not impose to many limitations. Each change script can have any number of SQL statements within it and must supply the necessary context, like database and schema names. The context can be supplied by using an explicit `USE <DATABASE>` command or by naming all objects with a three-part name (`<database name>.<schema name>.<object name>`). snowchange will simply run the contents of each script against the target Snowflake account, in the correct order.
```

Comment:
I might be missing something, but is this potentially confusing when combined with the `environment-name` parameter?

It looks like that parameter is only going to affect the naming of the metadata database, but until we get templating, scripts will always apply to the fixed database & schema regardless of environment setting.

Reply:
Yeah, I did think about that. Basically I changed the meaning of the `append-environment-name` parameter to append the environment name to the metadata database not the user created databases. Even with user templating/macros snowchange will always need to manage the change history table itself, and we'll need a way for users to specify which database/schema/table they want to store the changes in.

The other way to approach this, which might be more flexible would be to remove the `environment-name` and `append-environment-name` parameters entirely and allow the user to specify/override the change history table location. It looks like flyway supports this with a configuration setting where the user can explicitly define the table name. We'd probably need to allow setting/overriding any of the database/schema/table names.

Which option do you like better? Option #2 would make snowchange even simpler which is always nice (as we're not using the environment name for anything else, like you noticed).

Reply:
Yeah I like option 2, would be much clearer to a new user